tests/e2e/networks.json

```diff
@@ -0,0 +1,19 @@
+[
```

Comment:
I propose a structure:

```
{
  networkdID: {
    ...networkConfig
  }
}
```

so we can access the network configs directly by id